<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClubsEditor</class>
 <widget class="QWidget" name="ClubsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addClubBtn">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeClubBtn">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="clubList"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTabWidget" name="competitorList">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="ClubEditor" name="clubEditor">
       <attribute name="title">
        <string>Club Info</string>
       </attribute>
      </widget>
      <widget class="CompetitorList" name="competitorsList">
       <attribute name="title">
        <string>Competitors</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClubEditor</class>
   <extends>QWidget</extends>
   <header location="global">ClubEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompetitorList</class>
   <extends>QWidget</extends>
   <header>CompetitorList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
